src/libraries/System.Net.Requests/src/System/Net/Authorization.cs

```diff
@@ -15,24 +15,24 @@ public class Authorization
         public Authorization(string token, bool finished) :
             this(token, finished, null) { }
 
-        public Authorization(string token, bool finished, string connectionGroupId) :
+        public Authorization(string token, bool finished, string? connectionGroupId) :
             this(token, finished, connectionGroupId, false) { }
 
-        internal Authorization(string token, bool finished, string connectionGroupId, bool mutualAuth)
+        internal Authorization(string token, bool finished, string? connectionGroupId, bool mutualAuth)
         {
             Message = string.IsNullOrEmpty(token) ? null : token;
```

Comment:
Isn't this mean `token` can be null here and all above overloads?

Reply:
Sure. Changed.  The docs don't say, and no one uses these types, but seems fine to make it nullable.  Thanks.